src/main/java/org/sunbird/profile/service/UserBulkUploadService.java

```diff
@@ -141,57 +141,61 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                     if (nextRow.getCell(0) == null || nextRow.getCell(0).getCellType() == CellType.BLANK) {
                         errList.add("Full Name");
                     } else {
-                        userRegistration.setFirstName(nextRow.getCell(0).getStringCellValue());
+                        userRegistration.setFirstName(nextRow.getCell(0).getStringCellValue().trim());
                         if (!ProjectUtil.validateFullName(userRegistration.getFirstName())) {
                             invalidErrList.add("Invalid Full Name");
                         }
                     }
                     if (nextRow.getCell(1) == null || nextRow.getCell(1).getCellType() == CellType.BLANK) {
                         errList.add("Email");
                     } else {
-                        userRegistration.setEmail(nextRow.getCell(1).getStringCellValue());
+                        userRegistration.setEmail(nextRow.getCell(1).getStringCellValue().trim());
                     }
                     if (nextRow.getCell(2) == null || nextRow.getCell(2).getCellType() == CellType.BLANK) {
                         errList.add("Phone");
                     } else {
                         if (nextRow.getCell(2).getCellType() == CellType.NUMERIC) {
                             phone = NumberToTextConverter.toText(nextRow.getCell(2).getNumericCellValue());
-                            userRegistration.setPhone(phone);
+                            userRegistration.setPhone(phone.trim());
+                        }
+                        else if (nextRow.getCell(2).getCellType() == CellType.STRING)
```

Comment:
what if cellType is not numeric / String ??
need to throw error in that case - so that we know why it is failed...